boards/splitkb/aurora/lily58/README.md

```diff
@@ -0,0 +1,31 @@
+![Aurora Lily58](https://i.imgur.com/PNRXfCUh.jpg)
+
+# Aurora Lily58
+The Aurora Lily58 is a redesign by splitkb.com of the popular Lily58 by Naoki Katahira.
+
+Keyboard Maintainer: [splitkb.com](https://github.com/splitkb)  
+Hardware Supported: Pro Micro 5V/16MHz and compatible.  
+Hardware Availability: [splitkb.com](https://splitkb.com).
+
+# Assembly
+A build guide for the entire Aurora series is available at [docs.splitkb.com](https://docs.splitkb.com).
+
+# Features
+The Aurora Lily58 supports the following features:
+
+- Support for MX or Kailh Choc (v1 sold at splitkb.com, not v2) switches;
+- Top mounted controllers for a lower profile;
+- Up to two 128×32 pixel OLED displays;
+- Up to one EC11 rotary encoder per half in one of four possible positions;
+- Per-key RGB backlight by individual RGB LEDs;
+- Underglow by individual RGB LEDs;
+- Support for a power switch for wireless controllers;
+- Support for the splitkb.com tenting puck.
+
+# What is the Aurora Series?
+
+The Aurora series were designed as a homage to open source keyboard kits that have carved their space into the community. Kits that provided something new and unique, kits that resonated with the community and that have both received and have given a lot of love in return.
+
+With the Aurora series, we aim to make these kits available to the community in a way that both honors the original, while also offering the best possible feature set and a consistent, well-documented build experience.
+
+For each kit sold, we give back to the community by donating €1 across [the firmware projects we support](https://opencollective.com/splitkb). This way we support open source projects and benefit everyone using keyboards, whether they use our kits or their own.
```

Comment:
Kinda odd to have this link when KMK isn't listed there.

Reply:
This is just a copy of the Readme from the QMK or VIAL repo I think. They don't just sell the keyboards with firmware made by the community, they also actually do make financial contributions (and I'm assuming code contributions). It speaks to their motivation. But there are only so many firmwares they can contribute to themselves, hence this community contribution. If KMK is made compatible using this, it might be the first step to have KMK on that list?

I don't have a horse in this race. From my perspective it's factually correct information. But if it needs to be removed, I'll do just that.

Reply:
The "we" in that section is definitely misleading. Also, if you copy something verbatim, you have to cite the source.

Reply:
I've rewritten parts of the readme to have it not be a verbatim copy anymore (apart from the board features). Doing so I hope I tackled most or all of the remarks given about the readme.